Dokan/Dokan/Scenes/ProductDetails/TitleQuantity/ProductTitleQuantityView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ProductTitleQuantityView.swift
+//  Dokan
+//
+//  Created by ziad on 21/07/2022.
+//
+
+import UIKit
+class ProductTitleQuantityView: UIView {
+    // MARK: Outlet
+    
+    @IBOutlet private(set) weak var productTitle: UILabel!
+    @IBOutlet private(set) weak var productCurrency: UILabel!
+    @IBOutlet private(set) weak var productPrice: UILabel!
+    @IBOutlet private(set) weak var productReviewAverage: UILabel!
+    @IBOutlet private(set) weak var productReviewCount: UILabel!
+    @IBOutlet private(set) weak var productStockBackgroundView: UIView!
+    @IBOutlet private(set) weak var productStockCount: UILabel!
+    
+    // MARK: Init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        stylingProductDetails()
+    }
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        stylingProductDetails()
+    }
+    
+    // MARK: Style
+
+    func stylingProductDetails() {
+        self.productStockBackgroundView.layer.cornerRadius = 10
+    }
+
+    func changeStockBackGroundColor(stockCount: Int) {
+        if stockCount > 10 {
+            self.productStockBackgroundView.backgroundColor = .green
+        } else if stockCount > 0 && stockCount <= 10 {
+            self.productStockBackgroundView.backgroundColor = .orange
+        } else {
+            self.productStockBackgroundView.backgroundColor = .red
+        }
+    }
+
+    // MARK: Configuration
+    
+    func passProductDetailsDataToView(productDetailsData: ProductDetails) {
+        self.productTitle.text = productDetailsData.productTitle
+        self.productCurrency.text = productDetailsData.productCurrency
+        self.productPrice.text = productDetailsData.productPrice
+        self.productReviewAverage.text = "\(productDetailsData.productReviewAverage)"
+        self.productReviewCount.text = "\(productDetailsData.productReviewCount)"
+        self.productStockCount.text = "\(productDetailsData.productStockCount)"
+    }
+}
+
+struct ProductDetails {
+    var productTitle: String
+    var productCurrency: String
+    var productPrice: String
+    var productReviewAverage: Double
+    var productReviewCount: Int
+    var productStockCount: Int
+}
```

Comment:
- A good approach is to add a view model as a nested type to the view. 
- Remove duplicates even in a properties names. 
- Immutable properties are preferred. 
- 
```suggestion
// MARK: - View Model
//
extension ProductTitleQuantityView { 
    struct ViewModel {
        let title: String
        let currency: String
        let price: String
        let reviewAverage: Double
        let reviewCount: Int
        let stockCount: Int
    }
}
```